mizar/obj/endpoint.py

```diff
@@ -130,6 +137,9 @@ def set_obj_spec(self, spec):
         self.cnidelay = get_spec_val('cnidelay', spec)
         self.provisiondelay = get_spec_val('provisiondelay', spec)
         self.pod = get_spec_val('pod', spec)
+        self.ingress_networkpolicies = get_spec_val('ingress_networkpolicies', spec)
+        self.egress_networkpolicies = get_spec_val('egress_networkpolicies', spec)
+        self.data_for_networkpolicy = get_spec_val('data_for_networkpolicy', spec)
```

Comment:
Same as above

Reply:
Yes, I added in Endpoint Constructor (init) function.

Reply:
Make sure it's added successfully by logging it in the workflow. Like, logger.info("Spec: ".format(ep.get_obj_spec())). I am not sure if adding in init will actually successfully add to Spec. If I remember correctly, spec is determined by CRD

Please reference : https://github.com/CentaurusInfra/mizar/blob/a0c8fae4a49734c424540c3dc5d3069cce70f23e/mizar/obj/endpoint.py#L62

I dont think get_spec_val will work for network policy because it's not part of CRD

Reply:
Removed from spec related.